pipeline/trusted-resources/pkg/trustedtask/verify.go

```diff
@@ -33,13 +33,13 @@ import (
 
 const (
 	signingConfigMap = "config-trusted-resources"
-	kmsAnnotation    = "tekton.dev/kms"
+	KmsAnnotation    = "tekton.dev/kms"
 )
 
-func verifier(ctx context.Context, annotations map[string]string) (signature.Verifier, error) {
-	if annotations[kmsAnnotation] != "" {
+func verifier(ctx context.Context, kmsPath string) (signature.Verifier, error) {
```

Comment:
Do we expect we'll only use kms for signing in the future? If not, it might be better to generalize this param, make it not specific to kmsPath

Reply:
Thanks! I found some issue of this current approach just this afternoon, will fix it later

Reply:
fixed in the latest commit, for we use configmap to config all keys instead of passing from tasks..